packages/react-native-lockdown/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,68 @@
+# Troubleshooting
+
+> [!IMPORTANT]
+>
+> Always execute `react-native start --reset-cache` or `react-native bundle --reset-cache` after Babel or Metro config changes to remove cached files.
+>
+> This will ensure a fresh build/bundle with your configuration changes, rather than relying on hot-reload or hot/cold starts.
+
+## `ReferenceError: Property 'require' doesn't exist`
+
+![ReferenceError: Property 'require' doesn't exist](assets/ReferenceError-Property-require-doesnt-exist.png)
+
+Upon inspecting the bundle, the reference error is throwing early when evaluating the SES shim.
+
+It is clear Babel is preparing it to be [transformed][ses-transformed-ext]:
+
+```js
+// http://localhost:8081/index.bundle//&platform=android
+// ...
+(function (global) {
+  var _asyncToGenerator = require("@babel/runtime/helpers/asyncToGenerator"); // <----- ERROR HERE
+  var _toArray = require("@babel/runtime/helpers/toArray");
+  var _objectWithoutProperties = require("@babel/runtime/helpers/objectWithoutProperties");
+  var _toConsumableArray = require("@babel/runtime/helpers/toConsumableArray");
+  var _slicedToArray = require("@babel/runtime/helpers/slicedToArray");
+  var _defineProperty = require("@babel/runtime/helpers/defineProperty");
+  var _objectDestructuringEmpty = require("@babel/runtime/helpers/objectDestructuringEmpty");
+  var _excluded = ["name", "message", "errors", "cause", "stack"],
+    _excluded2 = ["random"],
+    _excluded3 = ["__options__"],
+    _excluded4 = ["errorTaming", "errorTrapping", "reporting", "unhandledRejectionTrapping", "regExpTaming", "localeTaming", "consoleTaming", "overrideTaming", "stackFiltering", "domainTaming", "evalTaming", "overrideDebug", "legacyRegeneratorRuntimeTaming", "__hardenTaming__", "dateTaming", "mathTaming"];
+  function _defineAccessor(e, r, n, t) { var c = { configurable: !0, enumerable: !0 }; return c[e] = t, Object.defineProperty(r, n, c); }
+  // ses@1.13.0
+  (function (functors) {
+// ...
+```
+
+Assert Babel is configured to ignore your flavour(s) of SES shim (see the [ReadMe Babel config][readme-babel-config]) examples, then re-run React Native CLI with the `--reset-cache` flag.
+
+Now it should look much cleaner:
+
+```js
+// http://localhost:8081/index.bundle//&platform=android
+// ...
+(function (global) {
+  // ses@1.13.0
+  (functors => options => {
+// ...
+```
+
+## `Cannot read property 'setGlobalHandler' of undefined`
+
+![Cannot read property 'setGlobalHandler' of undefined](assets/Cannot-read-property-setGlobalHandler-of-undefined.png)
+
+It looks like React Native is having trouble [initializing][init-core-ext] when setting up [error handling][set-up-error-handling-ext], due to the missing [error guard][rn-js-polyfills-error-guard-ext] polyfill.
+
+Did you remember to include the React Native JS polyfills in your custom Metro config? See the [ReadMe Metro Existing config][readme-metro-config] example.
+
+## License
```

Comment:
I think this part is a leftover from when this content was in README. Troubleshooting docs don't need their own license ;)

Reply:
updated ^